src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: article
+title: Keyboard
```

Comment:
I think its better to stick with the product name and call this title "The Appwriter"

Reply:
Or  is this guide supposed to be for keyboards and not the Appwriter specifically ?

Reply:
Yes, we had everything combined so it was Keyboard, but now it makes sense to have the Appwriter page's title be "The Appwriter".